.claude/skills/skill-creator/scripts/utils.py

```
@@ -0,0 +1,47 @@
+
+from pathlib import Path
+
+
+
+def parse_skill_md(skill_path: Path) -> tuple[str, str, str]:
+    """Parse a SKILL.md file, returning (name, description, full_content)."""
+    content = (skill_path / "SKILL.md").read_text()
+    lines = content.split("\n")
+
+    if lines[0].strip() != "---":
+        raise ValueError("SKILL.md missing frontmatter (no opening ---)")
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)